### PR TITLE
Image Generator Naming Convention Enhancement

### PR DESCRIPTION
**Pull request by CalCameron**
Improved the image gen code to ensure a better quality image name, which is also unique. 

Resolved an issue with the file logger code I had, to ensure we append if we already have written.

### PR Quality Checklist
- [X] My pull request is atomic and focuses on a single change.
- [X] I have thoroughly tested my changes with multiple different prompts.
- [X] I have considered potential risks and mitigations for my changes.
- [X] I have documented my changes clearly and comprehensively.
- [X] I have not snuck in any "extra" small tweaks changes

### CODE REVIEW

File: autogpt/commands/image_gen.py [hunk 1]
```diff
@@ -22,7 +22,11 @@ def generate_image(prompt: str) -> str:
     Returns:
         str: The filename of the image
     """
-    filename = f"{str(uuid.uuid4())}.jpg"
+    ts =  time.time()
+    """First 3 words for the file name"""
+    split = prompt.split()[0:3]
+    name  = "-".join(split)
+    filename = name + str(ts) + ".jpg"
```
Comment (hdkiller) [inline autogpt/commands/image_gen.py:29]: Why don’t you let the model decide the filename instead of choosing it from the prompt which used to generate it? 

Add <filename> to the prompt and you will end up with more quality names.

Comment (CalCameron) [reply autogpt/commands/image_gen.py:29]: For now, I would feel this would work fine. It's done well

Comment (CalCameron) [reply autogpt/commands/image_gen.py:29]: It's better than the way it was before

File: autogpt/commands/file_operations.py [hunk 2]
```diff
@@ -133,7 +133,8 @@ def write_to_file(filename: str, text: str) -> str:
         str: A message indicating success or failure
     """
     if check_duplicate_operation("write", filename):
-        return "Error: File has already been updated."
+        append_to_file(filename, text)
+        return "Append to file complete!"
```
Comment (nponeccop) [inline autogpt/commands/file_operations.py:137]: This isn't related to a naming convention, is it? Submit as a separate PR

**Review by nponeccop — CHANGES_REQUESTED** [review]

File: autogpt/commands/file_operations.py (hunk 2 above)
Comment (CalCameron) [reply autogpt/commands/file_operations.py:137]: I added it in the description 'Resolved an issue with the file logger code I had, to ensure we append if we already have written.'

File: autogpt/commands/image_gen.py (hunk 1 above)
Comment (BillSchumacher) [reply autogpt/commands/image_gen.py:29]: Should have an underscore between the time and name though, also let's make this an fstring.

**Review by BillSchumacher — CHANGES_REQUESTED** [review]

**Comment by Pwuts** [comment]
Closing because not updated
